docs/user-guide/api-mediation/configuration-extender-passtickets.md

```diff
@@ -96,7 +158,16 @@ PERMIT IRRPTAUTH.<applid>.* CL(PTKTDATA) ID(<zowesrv>) ACCESS(UPDATE)
 SETROPTS RACLIST(PTKTDATA) REFRESH
 ```
 
-# Adding custom HTTP Auth headers to store user ID and PassTicket
+### Validate if the PassTicket Application is created
+
+```
+RLIST APPL APPLNAME ALL
```

Comment:
Same bug here - should it be <applid> instead of `APPLNAME`

Reply:
yes we can generalize it as "applid" for this whole document